docs/redoc/v0.4.2/openapi.json

```diff
@@ -1737,7 +1737,7 @@
     }
   },
   "externalDocs": {
-    "description": "Find out more about Qdrant aplications and demo",
+    "description": "Find out more about Qdrant applications and demo",
```

Comment:
I can't find any other source for this text... https://grep.app/search?q=Find%20out%20more%20about%20Qdrant%20aplications%20and%20demo
https://github.com/search?q=Find+out+more+about+Qdrant+aplications+and+demo+NOT+repo%3Aqdrant%2Fqdrant+NOT+is%3Afork&type=code